examples/step-75/step-75.cc

```diff
@@ -972,7 +972,7 @@ namespace Step75
     // The next part is going to be tricky. During execution of refinement, a
     // few hp-algorithms need to interfere with the actual refinement process on
     // the Triangulation object. We do this by connecting several functions to
-    // Triangulation::Signals: signals will be called at different stages during
+    // Triangulation::Signals::signals will be called at different stages during
```

Comment:
I would leave it as before so that `doxygen` is able to cross-reference. I address that in a follow-up.

Reply:
oops!